src/app/ledger_export_bench/README.md

```diff
@@ -0,0 +1,23 @@
+Ledger export benckmark
+=======================
+
+This package provides a couple of simple benchmarks for the most
+resource-consuming parts of the ledger export process, namely:
+* serialising and de-serialising a large runtime config;
+* converting accounts between runtime config format and ledger format.
+
+These benchmarks depend on a sufficiently large runtime config file
+supplied from outside, because creating one within a benchmark,
+while useful, is too involved, as it basically requires connecting
+to a live network. Besides, such a benchmark would be unreliable,
+as its results wouldn't be reproducible.
+
+Sadly, a runtime config containing realistic mainnet data is too
```

Comment:
Another alternative is to download ledger files from cloud storage. This could remedy the requirement of passing around ledger files between engineers. It is not a blocker in this case, but may be helpful in a future revision.